pdctl/command/store_command.go

```diff
@@ -47,6 +49,16 @@ func NewDeleteStoreCommand() *cobra.Command {
 	return d
 }
 
+// NewLabelStoreCommand returns a label subcommand of storeCmd.
+func NewLabelStoreCommand() *cobra.Command {
+	l := &cobra.Command{
+		Use:   "label <store_id> <key> <value>",
+		Short: "set a store's label value",
```

Comment:
I suppose no need to mind the difference of set and add. And the `store id` api will show labels, so seems unnecessary to add a `get` neither.